src/components/Cards/card-data/full-node-cards.js

```diff
@@ -0,0 +1,46 @@
+const fullNodeCards = [
+  {
+    href: '/full-node/run-a-full-terra-node',
+    lightSvg: '/img/icons/node.svg',
+    darkSvg: '/img/icons/dark/node.svg',
+    header: 'Run a full node',
+    summary: 'Walk through the configuration and management of a full node on Terra.'
```

Comment:
The phrasing used in other parts of the docs is "a full Terra node." 

So I would change this to "Walk through the configuration and management of a full Terra node."

or simply "Configure and manage a full Terra node."

LMK your thoughts on this.